### PR TITLE
Add playbooks to deploy VM's 

### PR DESCRIPTION
**Pull request by Paullgk**
This PR adds following files:

Playbooks:

- `vm_deploy.yaml`: a playbook to create and start a VM in cluster. VM image source file must be put in `vm_images/` directory.  The VM is identified by an unique id set by `vm_id` variable. VM image source file must be set using `image_name` variable.

- `vm_remove.yaml`: a playbook to stop and remove a VM in cluster. The VM is identified by an unique id set by `vm_id` variable.

Configuration templates: 

- `votp_vm_standard_config.xml.j2`: a standard configuration based on `votp_vm_rt_isolated.xml.j2` but using 1Gb of RAM.

### CODE REVIEW

File: templates/vm/votp_vm_standard_config.xml.j2 [hunk 1]
```diff
@@ -0,0 +1,66 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- Copyright (C) 2021, RTE (http://www.rte-france.com) -->
+<!-- SPDX-License-Identifier: Apache-2.0 -->
+
+<domain type="{{ vm_domaine | default('kvm') }}">
```
Comment (deribaucourt) [inline templates/vm/votp_vm_standard_config.xml.j2:5]: vm_domain?

File: playbooks/vm_remove.yaml [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+# Ansible playbook that remove a VM.
+---
+
+- hosts: hypervisors
+  name: Stop and remove guest{{ vm_id }}
+  vars:
+    - vm_id: 1
+# Create and start a VM
+  tasks:
+    - name: Stop VM
```
Comment (dupremathieu) [inline playbooks/vm_remove.yaml:13]: You can remove the VM without explicitly stop it (the removing action will stop the VM)

File: templates/vm/votp_vm_standard_config.xml.j2 [hunk 3]
```diff
@@ -0,0 +1,66 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- Copyright (C) 2021, RTE (http://www.rte-france.com) -->
```
Comment (dupremathieu) [inline templates/vm/votp_vm_standard_config.xml.j2:2]: 2023

File: templates/vm/votp_vm_standard_config.xml.j2 [hunk 4]
```diff
@@ -0,0 +1,66 @@
...
+                <model name="isa-serial" />
+            </target>
+        </serial>
+        <console type="pty">
+            <target type="serial" port="0" />
+        </console>
+        <memballoon model="virtio">
+	    <stats period="5" />
+        </memballoon>
+	<vsock model='virtio'>
```
Comment (dupremathieu) [inline templates/vm/votp_vm_standard_config.xml.j2:61]: wrong indentation

File: playbooks/vm_remove.yaml [hunk 5]
```diff
@@ -0,0 +1,22 @@
+# Copyright (C) 2023, RTE (http://www.rte-france.com)
+# SPDX-License-Identifier: Apache-2.0
+
+# Ansible playbook that remove a VM.
```
Comment (deribaucourt) [inline playbooks/vm_remove.yaml:4]: removes

File: playbooks/vm_deploy.yaml [hunk 6]
```diff
@@ -0,0 +1,40 @@
+# Copyright (C) 2023, RTE (http://www.rte-france.com)
+# SPDX-License-Identifier: Apache-2.0
+
+# Ansible playbook that create and start a VM.
```
Comment (deribaucourt) [inline playbooks/vm_deploy.yaml:4]: creates and starts

File: playbooks/vm_deploy.yaml [hunk 7]
```diff
@@ -0,0 +1,40 @@
...
+        system_image: /tmp/{{ image_name }}.qcow2
+        xml: >-
+          {{ lookup('template', xml_template,
+          template_vars=dict(
+            title='guest{{ vm_id }}',
+            ovs_port='VM{{ vm_id }}.0',
+            cpuset='{{ cpu_set }}',
+            mac_address='52:54:00:42:ab:0{{ vm_id }}', ),
+          errors='strict') }}
+        metadata:
```
Comment (deribaucourt) [inline playbooks/vm_deploy.yaml:38]: Is this used?

File: playbooks/vm_deploy.yaml [hunk 8]
```diff
@@ -0,0 +1,40 @@
...
+
+---
+
+# Fetch and create VM config file
+- hosts: hypervisors
+  name: Create and start guest{{ vm_id }}
+  vars:
+    - image_name: vm
+    - vm_id: 3
+    - cpu_set: 5
```
Comment (dupremathieu) [inline playbooks/vm_deploy.yaml:14]: I preferred using the cpu 6 for cpu_set. Because the 5 is not inside the RT slice in all our machines.

**Review by dupremathieu — CHANGES_REQUESTED** [review]

**Review by deribaucourt — CHANGES_REQUESTED** [review]

File: templates/vm/votp_vm_standard_config.xml.j2 (hunk 1 above)
Comment (Paullgk) [reply templates/vm/votp_vm_standard_config.xml.j2:5]: Nope, based on other VM configuration, it is `vm_domaine`

File: playbooks/vm_deploy.yaml (hunk 7 above)
Comment (Paullgk) [reply playbooks/vm_deploy.yaml:38]: Nope, I deleted it

File: playbooks/vm_remove.yaml [hunk 9]
```diff
@@ -0,0 +1,17 @@
...
+# SPDX-License-Identifier: Apache-2.0
+
+# Ansible playbook that removes a VM.
+---
+
+- hosts: hypervisors
+  name: Stop and remove guest{{ vm_id }}
+  vars:
+    - vm_id: 1
+# Create and start a VM
```
Comment (eroussy) [inline playbooks/vm_remove.yaml:11]: wrong comment

**Review by eroussy — CHANGES_REQUESTED** [review]

**Review by eroussy — APPROVED** [review]

**Pull request merged**